honestcash-v2/src/app/app.module.ts

```diff
@@ -18,22 +18,15 @@ import { environment } from '../environments/environment';
 import { TokenInterceptor } from './core/http/token.interceptor';
 import { HTTP_INTERCEPTORS } from '@angular/common/http';
 import { AppEffects } from './core/store/app/app.effects';
-import {library} from '@fortawesome/fontawesome-svg-core';
-import {faEnvelope, faUser} from '@fortawesome/free-regular-svg-icons';
-import {faKey} from '@fortawesome/free-solid-svg-icons/faKey';
-import {faExclamationCircle} from '@fortawesome/free-solid-svg-icons/faExclamationCircle';
-import {faComments, faGlobe, faHeart, faRetweet, faShareAlt, faSpinner, faTerminal} from '@fortawesome/free-solid-svg-icons';
-import {faBitcoin} from '@fortawesome/free-brands-svg-icons';
 import {FontAwesomeModule} from './core/modules/font-awesome.module';
+import {BrowserAnimationsModule} from '@angular/platform-browser/animations';
 
 @NgModule({
   declarations: [
     AppComponent
   ],
   imports: [
-    BrowserModule.withServerTransition({
-      appId: 'honestcash-v2'
-    }),
```

Comment:
Why did we remove it?

Reply:
Angular threw an error saying BrowserAnimationsModule and BrowserModule cannot be imported both as AnimationsModule already has the BrowserModule inside but I added it below instead at the top, this seems to fix the issue